internal_worker.go

```diff
@@ -106,8 +107,11 @@ type (
 		// Defines how many concurrent executions for task list by this worker.
 		ConcurrentActivityExecutionSize int
 
-		// Defines rate limiting on number of activity tasks that can be executed per second.
-		MaxActivityExecutionsPerSecond float32
+		// Defines rate limiting on number of activity tasks that can be executed per MaxActivityExecutionRateRefreshDuration.
+		MaxActivityExecutionRate int
```

Comment:
This is only apply to activity task. For workflow task, we probably don't need to expose to user, for now we use hardcoded rate at 1000 per second.